test/OpenTelemetry.Instrumentation.AWS.Tests/TestAWSClientInstrumentation.cs

```diff
@@ -198,8 +199,8 @@ private void ValidateAWSActivity(Activity aws_activity, Activity parent)
 
     private void ValidateDynamoActivityTags(Activity ddb_activity)
     {
-        Assert.Equal("DynamoDBv2.Scan", ddb_activity.DisplayName);
-        Assert.Equal("DynamoDBv2", Utils.GetTagValue(ddb_activity, "aws.service"));
+        Assert.Equal("DynamoDB.Scan", ddb_activity.DisplayName);
```

Comment:
why this assert changed ?

Reply:
The [AWSServiceName tag is set](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/89ff089bf07fe8576e7cf2027a020a305537ad91/src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSTracingPipelineHandler.cs#L232) based on the result from calling [`AWSServiceHelper.GetAWSServiceName`](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/89ff089bf07fe8576e7cf2027a020a305537ad91/src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSTracingPipelineHandler.cs#L215).  The [implementation of `GetAWServiceName` was changed in this PR](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1572/files/31344ee2876fb471c3de5719afd3cad383c7a078#diff-f63d829e66f547c60534775fb2216f1fe15ea5e74d04b3233d540926ef29fc31L24) to use `ServiceMetaData.ServiceId` instead of `requestContext.Request.ServiceName`; moving the handler code earlier in the SDK Pipeline means the `requestContext.Request` object isn't yet available.

`ServiceId` and `ServiceName` are sometimes subtly different, as is the case with dynamo: **DynamoDB** vs **DynamoDBv2**.

I could see an argument that this is a breaking change - existing users would see the new tag value after upgrading.  To prevent it, I'd need to hardcode a mapping table for dynamodb and sns.  When I considered this, my opinion was the name change was acceptable and the maintenance overhead of the mapping table was not worth it.

But I'd very much welcome additional opinions, especially if CNCF has guidance around this issue.

Reply:
As you said, the name shouldn't change, while we can change the logic... But I see  [here](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/Instrumentation.AWS-1.1.0-beta.3/src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md#110-beta1) similar changes. Approving and let's mark this as breaking changes. @Kielek FYI..